### ISSUE TITLE
CSS-Klasse für alle Backend-Icons?

### ISSUE THREAD

**Issue by NinaG** [contao/contao#6368]
Für das Erstellen bzw. Modifizieren von Backend-Themes wäre es hilfreich, wenn sämtliche Icons im Backend ergänzend eine gemeinsame Klasse wie z.B. _tl_icon_ hätten. Nur als kleine Anregung :)

**Comment by NinaG** [comment]
Ich habe mir erstmal damit beholfen, könnte mir aber vorstellen, dass eine Klasse die Sache noch etwas erleichtern würde:
/* Icons standardmäßig größer erscheinen zu lassen, da das Backend primär von sehbehinderten Redakteuren genutzt wird
`img[src$=".svg"] { min-width: 24px; height: auto; }`

**Comment by aschempp** [comment]
I don't think this should be a class on the icon itself. In Contao 5.6, there should be a `.operations` class on the operations, you likely also want to increase the font size of the menu etc.

**Comment by zoglo** [comment]
> you likely also want to increase the font size of the menu etc.

I agree. The mentioned CSS `min-width: 24px` indicates that the target size of the operations itself is too small: https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html
